plugins/available/base.plugin.bash

```diff
@@ -88,16 +88,15 @@ function pmdown ()
       echo "You don't have a markdown command installed!"
     fi
 }
-
+#if using mkcd many dirs, it will make all dirs and cd into the lastest one.
 function mkcd ()
 {
     about 'make a directory and cd into it'
     param 'path to create'
     example '$ mkcd foo'
     example '$ mkcd /tmp/img/photos/large'
```

Comment:
Please add an additional example here showing how to use this with multiple directories